recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-core/images/core-image-thin-initramfs.bbappend

```diff
@@ -14,14 +14,15 @@ IMAGE_INSTALL_append = " \
     domd \
     domd-run \
     domd-install-artifacts \
+    ${@bb.utils.contains('XT_GUESTS_INSTALL', 'doma', 'u-boot', '', d)} \
 "
 
 XT_GUESTS_INSTALL ?= "doma domf"
 
 python __anonymous () {
     guests = d.getVar("XT_GUESTS_INSTALL", True).split()
     if "doma" in guests :
-        d.appendVar("IMAGE_INSTALL", " doma doma-run doma-install-artifacts")
+        d.appendVar("IMAGE_INSTALL", " doma doma-run")
```

Comment:
What if we do not want DomA to use u-boot?

Reply:
you mean, we may have a case, when 'u-boot into initrmfs' should not work for doma ?

Reply:
Hm, probably not for this product